src/codec/binflux.rs

```diff
@@ -42,18 +42,18 @@ impl BInflux {
 
         let mut res = Vec::with_capacity(512);
         res.write_u16::<BigEndian>(0)?;
-        if let Some(measurement) = v.get("measurement").and_then(Value::as_str) {
+        if let Some(measurement) = v.get_str("measurement") {
```

Comment:
aaahhh, so good!

Reply:
Ja, I had to write so many of those `and_then` that I realized that is something missing in the value-trait.

Reply:
💅  That is sweetness itself!